includes/class-wp-job-manager-shortcodes.php

```diff
@@ -766,6 +772,9 @@ public function output_jobs( $atts ) {
 		if ( ! empty( $atts['post_status'] ) ) {
 			$data_attributes['post_status'] = implode( ',', $atts['post_status'] );
 		}
+		if ( ! empty( $atts['featured_first'] ) ) {
+			$data_attributes['featured_first'] = $atts['featured_first'] ? 'true' : 'false';
+		}
```

Comment:
Is it really what you wanted to do ?
If $atts['featured_first'] is false, then it's empty, which means $data_attributes won't be populated.
If it's what you wanted to do, then it's ok :)

Reply:
yeah no, not what I want to do.  Removed in c3b858a88193136c708f9508ab0ea779707195b0